Source/WebCore/page/Quirks.cpp

```diff
@@ -1542,6 +1542,15 @@ bool Quirks::shouldUseEphemeralPartitionedStorageForDOMCookies(const URL& url) c
     return false;
 }
 
+// rdar://155649992
+bool Quirks::shouldAllowDownloadsInSpiteOfCSP() const
+{
+    if (!needsQuirks())
+        return false;
+
+    return isDomain("dropbox.com"_s);
```

Comment:
It would be more efficient if you added a new flag to `QuirksData`, that is set in `Quirks::determineRelevantQuirks()` so that we do not have to do this string comparison every time we want to check if this Quirk is needed in the DOM.

Reply:
I verified that actually doesn't work because at this point in the loading process Quirks::determineRelevantQuirks just returns early.  As it is, this is only one comparison per navigation that we decide to download.

Reply:
Great! Then ship it!